include/grpc/grpc_crl_provider.h

```diff
@@ -38,32 +38,26 @@ class Crl {
 
  protected:
   Crl() = default;
-
- private:
-  std::string raw_crl_;
 };
 
-// Opaque representation of a Certificate
-class Cert {
+// Information about a certificate to be used to fetch its associated CRL.
+class CertificateInfo {
  public:
-  static absl::StatusOr<std::unique_ptr<Cert>> Parse(
-      absl::string_view cert_string);
-  virtual ~Cert() = default;
+  explicit CertificateInfo(absl::string_view issuer) : issuer_(issuer) {}
+  absl::string_view GetIssuer() { return issuer_; }
 
  protected:
```

Comment:
Don't need a `protected` section if there's nothing in the section.

Reply:
This has a protected constructor now that it's abstract

Reply:
No need to define the ctor at all; it will get one by default, since there are no other ctors defined.  And there's no need to make the ctor protected if the class has an abstract virtual method, because then there's no way for anything but a subclass to instantiate it anyway.

I think the entire `protected` section can be removed.